crates/engine/tree/src/tree/root.rs

```diff
@@ -229,7 +233,7 @@ impl ProofSequencer {
 
         // return early if we don't have the next expected proof
         if !self.pending_proofs.contains_key(&self.next_to_deliver) {
-            return Vec::new()
+            return Vec::new();
```

Comment:
```suggestion
            return Vec::new()
```
pls revert unrelated formatting changes. makes it hard to see the actual diff to review.